conjur-oss/UPGRADING.md

```diff
@@ -511,7 +511,7 @@ $  namespace="<conjur-namespace>"
 
 $  helm_chart_name=$(helm list -n "$namespace" -q)
 $  account=$(helm show values "$helm_chart_name" | \
-             awk '/^account:/{print $2}' | \
+             awk '/^account.name:/{print $2}' | \
```

Comment:
The `.` needs to be escaped via `\.`, otherwise it'll match for any character, e.g.

```
echo "account@name: moo" | awk '/^account.name:/{print $2}'
```